content/en/docs/concepts/services-networking/_index.md

```diff
@@ -23,9 +21,14 @@ implementation (barring any intentional network segmentation policies):
    * agents on a node (e.g. system daemons, kubelet) can communicate with all
      pods on that node
 
-Note: For those platforms that support `Pods` running in the host network (e.g.
-Linux), when pods are attached to the host network of a node they can still communicate 
-with all pods on all nodes without NAT.
+{{< note >}}
+For those platforms that support `Pods` running in the host network (e.g.
+Linux):
+{{< /note >}}
+
+   * host-network pods of a node can connect directly to the pod IP's on all
+     nodes, however, unlike pod-network pods, the source IP address might not be
+     present in the `Pod` `status.PodIPs` field.
 
 This model is not only less complex overall, but it is principally compatible
```

Comment:
This model is not only less complex overall, but it is also principally compatible
with the desire for Kubernetes to enable low-friction porting of apps from VMs.

Reply:
You recommend omitting “to containers”?

Reply:
No!